### PR TITLE
Add @cjuniet's keymap/layout/userspace

### PR DESCRIPTION
**Pull request by cjuniet**
## Description

Add @cjuniet's userspace and keymap for Kyria keyboards.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* This firmware will show modifiers & locks states as a status bar on the master OLED display, with icons from a custom font:
IMG_20200228_210427

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kyria/keymaps/cjuniet/keymap.c [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS)
+};
+
+LEADER_EXTERNS();
+
+void keyboard_post_init_user(void) {
+    led_t led_state = host_keyboard_led_state();
+    if (!led_state.num_lock) {
+        register_code(KC_NUMLOCK);
+        unregister_code(KC_NUMLOCK);
```
Comment (fauxpark) [inline keyboards/kyria/keymaps/cjuniet/keymap.c]:
```suggestion
        tap_code(KC_NLCK);
```

Comment (fauxpark) [inline keyboards/kyria/keymaps/cjuniet/keymap.c]:
```suggestion
        tap_code(KC_NLCK);
```

File: users/cjuniet/glcdfont.c [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+
+#ifdef __AVR__
+#    include <avr/io.h>
+#    include <avr/pgmspace.h>
+#elif defined(ESP8266)
+#    include <pgmspace.h>
+#else
+#    define PROGMEM
+#endif
+
```
Comment (fauxpark) [inline users/cjuniet/glcdfont.c]:
```suggestion
#include "progmem.h"

```

File: users/cjuniet/cjuniet.h [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#ifndef USERSPACE
+#define USERSPACE
+
+#include "quantum.h"
+
+void render_status_bar(void);
+
+#endif
```
Comment (fauxpark) [inline users/cjuniet/cjuniet.h]:
```suggestion
```

File: users/cjuniet/cjuniet.h [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#ifndef USERSPACE
+#define USERSPACE
```
Comment (fauxpark) [inline users/cjuniet/cjuniet.h]:
```suggestion
#pragma once
```

**Comment by cjuniet** [comment]
@fauxpark Thanks for the review. I've applied all your requested changes.

File: keyboards/kyria/keymaps/cjuniet/keymap.c [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+    return update_tri_layer_state(state, _SYMBOLS, _NUMBERS, _SPECIALS);
+}
+
+void matrix_scan_user(void) {
+    LEADER_DICTIONARY() {
+        leading = false;
+        leader_end();
+
+        SEQ_ONE_KEY(KC_ESC) {
+            SEND_STRING(SS_LCTL(SS_LGUI("q")));
```
Comment (drashna) [inline keyboards/kyria/keymaps/cjuniet/keymap.c]: You could also do:
```suggestion
            tap_code16(LCTL(LGUI(KC_)));
```

However, this is mostly just semantics.

Comment (cjuniet) [reply keyboards/kyria/keymaps/cjuniet/keymap.c]: done

**Comment by noroadsleft** [comment]
For future reference, we recommend against committing to your `master` branch as you've done here, because pull requests from modified `master` branches can make it more difficult to keep your QMK fork updated. It is highly recommended for QMK development – regardless of what is being done or where – to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.

There are instructions on how to keep your fork updated here:

[**Best Practices: Your Fork's Master: Update Often, Commit Never**](https://docs.qmk.fm/#/newbs_git_using_your_master_branch)

[Part 3 of our Best Git Practices section](https://docs.qmk.fm/#/newbs_git_best_practices) walks you through how to resynchronize your branch. If you need any help with this just ask.

Thanks for your contribution!

**Pull request merged**